<launch>

  <group ns="strategy_level">

    <node pkg="strategy_discrete" name="TuLiPstrategy_server" type="TuLiPstrategy_server.py" output="screen" />
    <node pkg="strategy_discrete" name="TuLiPstrategy_server_2" type="TuLiPstrategy_server.py" args="TuLiPstrategy_server:=TuLiPstrategy_server_2 TuLiPstrategy:=TuLiPstrategy_2" output="screen" />
    <node pkg="strategy_discrete" name="strategy_wrapper_0" type="strategy_wrapper.py" args="0" output="screen" />
    <node pkg="strategy_discrete" name="strategy_wrapper_1" type="strategy_wrapper.py" args="1" output="screen" />
  
  </group>

  <node pkg="strategy_discrete" name="BluSTL_interface" type="BluSTL_interface.py" args="BluSTL_signal:=BluSTL_signal_0 Matlab_input:=Matlab_input_0 location_assignment:=strategy_level/location_assignment_0 Matlab_output:=Matlab_output_0" output="screen" />

  <node pkg="strategy_discrete" name="BluSTL_interface_2" type="BluSTL_interface.py" args="BluSTL_signal:=BluSTL_signal_1 Matlab_input:=Matlab_input_1 location_assignment:=strategy_level/location_assignment_1 Matlab_output:=Matlab_output_1" output="screen" />  

</launch>
